components/ThermostatScreen/ThermostatGauge.tsx

```diff
@@ -236,18 +237,22 @@ const styles = StyleSheet.create({
   indoorTemp: {
     ...typography.headline1Bold,
     color: theme.primary,
+    includeFontPadding: false,
+    paddingLeft: 14,
   },
   degree: {
     marginTop: 10,
-    fontSize: 18,
+    fontSize: 38,
     marginLeft: -2,
     color: theme.primary,
+    includeFontPadding: false,

```

Comment:
learned that this is needed for Android - if it's missing, there's extra padding for this font. https://reactnative.dev/docs/next/text-style-props#includefontpadding